content/doc/book/managing/nodes.adoc

```diff
@@ -45,9 +45,10 @@ clock time/sync and response time.
 A node is taken offline if any of these values
 go outside the configured threshold.
 
-The Jenkins controller itself runs on a special _built-in node_.
-It is possible to run agents and executors on this built-in node
-although this can degrade performance, reduce scalability of the Jenkins instance, and create serious security problems
+The Jenkins controller itself runs on a special _built-in node_. 
+The built-in node is a node inside the controller process. It is possible to run agents with it and it can have executors. 
+although Executors exist, are available or busy, and can be added or removed 
+but this can degrade performance, reduce scalability of the Jenkins instance, and create serious security problems
```

Comment:
Unclear how this relates to the start of the sentence.

Reply:
>Also, this content from the linked issue was meant mostly to describe reality, not be copy & pastable into the docs.
>
I dont know much about core jenkins internals so what should i use as reference ?
if there is a document can you give me the link,this would also increase my knowledge of jenkins internals

Reply:
> I dont know much about core jenkins internals so what should i use as reference ?
if there is a document can you give me the link,this would also increase my knowledge of jenkins internals

You're writing the documentation here. If we had it, it wouldn't need to be written 😉